torch/ao/quantization/_pt2e/decompositions.py

```diff
@@ -0,0 +1,29 @@
+# TODO: move _decomposed.py file to _pt2e folder
+import torch
+from torch.ao.quantization.fx._decomposed import quantized_decomposed_lib  # noqa: F401
+from torch._decomp import register_decomposition
+
+@register_decomposition(torch.ops.quantized_decomposed.quantize_per_tensor)
+def quantize_per_tensor(
```

Comment:
@SherlockNoMad one question I have is that if I should just reuse the functions defined in `torch.ao.quantization.fx._decmposed`, these functions have the same impl, but they have some additional checks

Reply:
Yes, looks very reasonable to me to just reuse.

Reply:
but then the additional checks will be traced as well? can we remove them?

Reply:
sorry, I'm just ramping on on decomps.  I think we should avoid copy-pasta here.  The naive solution would be to leave the checks in `torch/ao/quantization/fx/_decomposed.py` and have that function call here for the math.

Reply:
sure, I can do that